integration-tests/__tests__/coverage_transform_instrumented.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {readFileSync} = require('fs');
+const path = require('path');
+const SkipOnWindows = require('../../scripts/SkipOnWindows');
+const {cleanup, run} = require('../utils');
+const runJest = require('../runJest');
+
+const dir = path.resolve(__dirname, '../coverage-transform-instrumented');
+const coverageDir = path.join(dir, 'coverage');
+
+SkipOnWindows.suite();
```

Comment:
Can we try to remove all these references and see if AppVeyor is happy about it?

Reply:
By "all" I mean scoped to this PR of course :D

Reply:
Will do.

Reply:
Didn't work out. Broke the Appveyor https://ci.appveyor.com/project/Daniel15/jest/build/6781